pandas/tests/test_generic.py

```diff
@@ -415,6 +415,10 @@ def test_sample(self):
                 o.sample(frac=0.7, random_state=np.random.RandomState(test)),
                 o.sample(frac=0.7, random_state=np.random.RandomState(test)))
 
+            np.random.seed(test)
+            self._compare(o.sample(n=4), o.sample(n=4))
```

Comment:
This test doesn't look right to me. You need to call `np.random.seed(test)` before every call to `sample`, not just the first one.